api/v1/submissions/PKPSubmissionController.php

```diff
@@ -1558,6 +1559,15 @@ public function addContributor(Request $illuminateRequest): JsonResponse
 
         $author = Repo::author()->newDataObject($params);
         $newId = Repo::author()->add($author);
+
+        $params['id'] = $newId;
```

Comment:
should this be 'authorId' instead of 'id'?

Reply:
Here it is called 'id', it is however the author id.